http-server-netty/src/main/java/io/micronaut/http/server/netty/configuration/NettyHttpServerConfiguration.java

```diff
@@ -728,6 +729,39 @@ public void setJsonBufferMaxComponents(int jsonBufferMaxComponents) {
         this.jsonBufferMaxComponents = jsonBufferMaxComponents;
     }
 
+    /**
+     * Prior to 4.4.0, the Micronaut HTTP server used a multi-pipeline approach for handling HTTP/2
+     * connections where every request got its own netty pipeline with HTTP/2 to HTTP/1.1
+     * converters on the pipeline. This allowed for using mostly unchanged HTTP/1.1 in the request
+     * handling and any {@link io.micronaut.http.server.netty.NettyServerCustomizer}s.
+     * <p>
+     * As of 4.4.0, this approach was replaced with a more performant HTTP/2-specific
+     * implementation. This means worse compatibility with HTTP/1.1-based code paths and
+     * customizers, however. Setting this option to {@code true} returns to the old behavior.
```

Comment:
Does this mean that unless this is true, Http/1.1 services will notice a slowdown?

Are we going to keep this as `true` for the forseeable future? (I see the field is marked "TODO") 🤔

It's also something to rememeber if there's Http/2 benchmarks out there...